docs/comment_copy_guidelines.md

```diff
@@ -0,0 +1,133 @@
+# Comment Copy Guidelines
```

Comment:
Maybe I am missing some context but I ask myself every time what the meaning of this headline is. I also have the impression that people are confusing this to be a guideline for mentoring comments and start applying them to their personal mentoring style. With some points described here this might be good, but some should definitely not be applied to personal mentoring.

Suggestion: "Comment Guidelines for Auto-Mentoring"

Reply:
As a side note (but also maybe its the point), I think we need some general mentoring guidelines in another repo too.

Reply:
I'm meh on this, since this is the _automated mentoring_ repository; however if it helps at least 1 person, happy to change. How about one of these:

- Auto-Mentoring Comment Copy Guidelines
- Auto-Mentoring Comment Guidelines
- Comment Copy Guidelines for Auto-Mentoring
- Comment Guidelines for Auto-Mentoring
- something else?

I personally like some indication that this is about the commentary itself, not about comments in general, since _when_ to comment is covered in a different document.

Reply:
> Find something positive to say about their code if possible.

For what it's worth, I generally am _very_ positive when mentoring, because every submission is a step forward and I think encouragement is great, but saying something positive for the sake of something positive, in a similar manor as to how _some_ people (and companies) want PR reviews to go, is not something that is appreciated _in every culture_.

For example: In a large cross-section of my personal circles, empty positivity (like "The Praise Sandwich") for the sake of saying something positive, is considered **extremely rude** because it's ingenuous and completely removes meaning which detracts from anyone _actually being positive_.

Begin neutral is generally safer.